include/ibamr/INSHierarchyIntegrator.h

```diff
@@ -355,6 +355,11 @@ class INSHierarchyIntegrator : public IBTK::HierarchyIntegrator
      */
     int getNumberOfCycles() const override;
 
+    /*!
+     * Pure virtual method to project the velocity field following a regridding operation.
+     */
+    virtual void regridProjection() = 0;
```

Comment:
So, uhh, should this be `public` or `protected`? :-)

Reply:
:speak_no_evil:

Reply:
It seems like this is something that the solver can do as appropriate, but perhaps should not be done by external callers.

Reply:
I really did mean to make this protected, as the commit states. I fixed it.